src/main/java/kg/jarkyn/cobspecserver/Response.java

```diff
@@ -0,0 +1,13 @@
+package kg.jarkyn.cobspecserver;
+
+public class Response {
+    private String status;
+
+    public Response(String status) {
+        this.status = status;
+    }
+
+    public byte[] getBytes() {
```

Comment:
this is because bytes are already expected elsewhere to write to client stream, so not sure what you mean by too much of a detail
are you also referring to the naming?